components/match2/wikis/arenafps/get_match_group_copy_paste_wiki.lua

```diff
@@ -0,0 +1,83 @@
+---
+-- @Liquipedia
+-- wiki=arenafps
+-- page=Module:GetMatchGroupCopyPaste/wiki
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Table = require('Module:Table')
+
+--[[
+
+WikiSpecific Code for MatchList and Bracket Code Generators
+
+]]--
+
+local wikiCopyPaste = Table.copy(require('Module:GetMatchGroupCopyPaste/wiki/Base'))
```

Comment:
```suggestion
---@class ArenaFPSMatchCopyPaste: MatchCopyPaste
local WikiCopyPaste = Table.copy(require('Module:GetMatchGroupCopyPaste/wiki/Base'))
```

also in the commons module add `---@class MatchCopyPaste` as class declaration above line 17

Reply:
this means also adjusting the variable in the whole module 
also the change on commons wasn't done (yet)